js/base/common/ExchangeCommon.js

```diff
@@ -0,0 +1,322 @@
+'use strict';
+
+// ----------------------------------------------------------------------------------------------------------------------------
+// This file contains all methods part of the Exchange instance that do not have an exchange-specific implementation
+// thus can be transpiled to the other languages. Since they are part of the Exchange instance (at runtime),
+// they can make use of the 'this' and access everything that defined in the Exchange class (options, methods, etc).
+//
+// Warning: Every method here needs to have the 'function' identifier in the signature.
+// Warning: Do not declare classes or global variables in here
+// Warning: Every time a method is added, don't forget to add it to module.exports as well
+// -----------------------------------------------------------------------------------------------------------------------------
```

Comment:
@ttodua the methods to be transpiled can be added here, I've already added some so that I could make some tests